rest/admin_api_auth_test.go

```diff
@@ -508,6 +508,7 @@ func TestAdminAuthWithX509(t *testing.T) {
 	_, statusCode, err = checkAdminAuth("", "invalidUser", "invalidPassword", "", httpClient, managementEndpoints, true, []Permission{{"!admin", false}}, nil)
 	assert.NoError(t, err)
 	assert.Equal(t, http.StatusUnauthorized, statusCode)
+	require.Contains(t, err.Error(), ErrInvalidLogin.Message)
 }
 
 func TestAdminAPIAuth(t *testing.T) {
```

Comment:
There a new assertion for error message in this test from the original PR in this test